explorer/templatetags/explorer_tags.py

```diff
@@ -0,0 +1,19 @@
+from django import template
+from django.utils.module_loading import import_string
+
+from .. import app_settings
+
+
+register = template.Library()
+
+
+@register.inclusion_tag('explorer/export_buttons.html')
+def export_buttons(query=None):
+    exporters = {}
+    for key, value in app_settings.EXPLORER_DATA_EXPORTERS.items():
+        exporter_class = import_string(value)
+        exporters[key] = exporter_class.name
+    return {
+        'exporters': exporters,
+        'query': query,
+    }
```

Comment:
This is cool. From a UI perspective I was imagining a single 'Download' button, with a drop-down box to select the format, but let's give this a try. If it's clunky, maybe we revisit.